control-plane/Makefile

```diff
@@ -27,22 +27,22 @@ CONSUL_K8S_IMAGE_VERSION?=latest
 ci.dev-tree:
 	@$(SHELL) $(CURDIR)/build-support/scripts/dev.sh $(DEV_PUSH_ARG)
 
-# In CircleCI, the linux binary will be attached from a previous step at pkg/bin/linux_amd64/. This make target
+# In Github Actions, the linux binary will be attached from a previous step at pkg/bin/linux_amd64/. This make target
 # should only run in CI and not locally.
 ci.dev-docker:
 	@echo "Pulling consul-k8s container image - $(CONSUL_K8S_IMAGE_VERSION)"
 	@docker pull hashicorp/consul-k8s:$(CONSUL_K8S_IMAGE_VERSION) >/dev/null #todo change this back after pulling it the first time since the dockerfile is FROM this image
 	@echo "Building consul-k8s Development container - $(CI_DEV_DOCKER_IMAGE_NAME)"
 	@docker build -t '$(CI_DEV_DOCKER_NAMESPACE)/$(CI_DEV_DOCKER_IMAGE_NAME):$(GIT_COMMIT)' \
 	--build-arg CONSUL_K8S_IMAGE_VERSION=$(CONSUL_K8S_IMAGE_VERSION) \
-	--label COMMIT_SHA=$(CIRCLE_SHA1) \
-	--label PULL_REQUEST=$(CIRCLE_PULL_REQUEST) \
-	--label CIRCLE_BUILD_URL=$(CIRCLE_BUILD_URL) \
+	--label COMMIT_SHA=$(GITHUB_SHA) \
+	--label PULL_REQUEST=$(GITHIB_PULL_REQUEST) \
+	--label GITHUB_BUILD_URL=$(GITHUB_SERVER_URL)/$(GITHUB_REPOSITORY)/actions/runs/$(GITHUB_RUN_ID) \
```

Comment:
Should we revert this change since we're keeping `dev-upload-docker` in circle for now?  Maybe we can define another makefile target so we don't forget how to define these correctly?

Reply:
Good catch! I have reverted, added another make target `ci.dev-docker-github` and updated the commented out section of build-and-test.yml